public/js/main.js

```diff
@@ -1,22 +1,124 @@
 'use strict';
-
-// Alert user about comment submission success
-// Get the parameter from the URL
-// Disable linter for this line because web environment
-// isn't disabling node checks
-const urlParams = new URLSearchParams(window.location.search);	// eslint-disable-line node/no-unsupported-features/node-builtins
-const msg = urlParams.get('message');
+// function for toggling modal
 const modal = document.querySelector('.modal');
 const toggleModal = () => {
 	modal.classList.toggle('show-modal');
-};
+}
+// event listener for clicking out of modal
 const windowOnClick = (event) => {
 	if (event.target === modal) {
 		toggleModal();
 	}
-};
-if (msg) {
-	window.addEventListener('click', windowOnClick);
-	document.getElementById('modal-text').innerText = msg;
-	toggleModal();
-}
\ No newline at end of file
+}
+window.addEventListener('click', windowOnClick);
+
+// Handle form submit
+const commentForm = document.getElementById('commentForm');
+
+commentForm.addEventListener('submit', function (e) {
+	e.preventDefault();
+	const formData = new FormData(this);
+	const searchParams = new URLSearchParams();
+
+	for (const pair of formData) {
+		searchParams.append(pair[0], pair[1]);
+	}
```

Comment:
This still doesn't make sense to me. Form data should serialize just fine on its own. Worth looking into. Maybe file a bug and put a `// TODO(gh##)` comment here?

Reply:
I installed some parsing middleware so that express can parse multipart/form-data and now I don't have to do this searchParams thing anymore.